maths/area.py

```diff
@@ -458,6 +458,49 @@ def area_rhombus(diagonal_1: float, diagonal_2: float) -> float:
     return 1 / 2 * diagonal_1 * diagonal_2
 
 
+def area_reg_polygon(sides: int, length: float) -> float:
+    """
+    Calculate the area of a regular polygon.
+    Wikipedia reference: https://en.wikipedia.org/wiki/Polygon#Regular_polygons
+    Formula: (n*s^2*cot(pi/n))/4
+    >>> area_reg_polygon(3, 10)

```

Comment:
```suggestion

    >>> area_reg_polygon(3, 10)
```

Reply:
Thank you @CaedenPH for your suggestion.